bittensor/commands/schedule_coldkey_swap.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import sys
+from typing import List, Optional, Tuple, Union
+
+from rich.prompt import Confirm, Prompt
+from tqdm import tqdm
+
+import bittensor
+from bittensor.utils.balance import Balance
+
+from . import defaults
+from .utils import get_hotkey_wallets_for_wallet
+
+console = bittensor.__console__
+
+
+class ScheduleColdKeySwapCommand:
+    """
+    Executes the ``schedule_coldkey_swap`` command to schedule a coldkey swap on the Bittensor network.
```

Comment:
This command ostensibly schedules a swap of the coldkey, but when? As a user, I have no idea when/where this will occur.